lib/omnibus/software.rb

```diff
@@ -833,17 +833,17 @@ def version_guid
       fetcher.version_guid
     end
 
+    def resolved_version
```

Comment:
A little hesitant to call this resolved version because there is a resolved_version in the fetcher as well.